jsbi.mjs

```diff
@@ -1905,4 +1905,13 @@ JSBI.__kBitConversionBuffer = new ArrayBuffer(8);
 JSBI.__kBitConversionDouble = new Float64Array(JSBI.__kBitConversionBuffer);
 JSBI.__kBitConversionInts = new Int32Array(JSBI.__kBitConversionBuffer);
 
+// For IE11 compatibility.
```

Comment:
Do we want to mention in a comment that the fallback helper functions are simplified versions, and thus not re-usable as polyfills?

Reply:
Done.